artifacts/scripts/construct.sh

```diff
@@ -141,18 +141,4 @@ PUSH_SCRIPT=../push-tags-${REPO}-${DST_BRANCH}.sh
 echo "#!/bin/bash" > ${PUSH_SCRIPT}
 chmod +x ${PUSH_SCRIPT}
 
-if [ -z "${SKIP_TAGS}" ]; then
```

Comment:
why not just disable this block?

Reply:
Do you mean comment it out? I figured we would never want to run it from the godeps branch so we could just remove it.